### PR TITLE
fix: is -> if and alphabetize keywords 

### PR DESCRIPTION
**Pull request by willemneal**

### CODE REVIEW

**Comment by willemneal** [comment]
@saulecabrera I don't have permission to push to your branch, but here are my edits.

File: crates/grammar/meta/syntax.toml [hunk 1]
```diff
@@ -102,6 +102,10 @@ supported = true
 name = "get"
 supported = true
 
+[[keywords]]
+name = "if"
+supported = false
```
Comment (willemneal) [inline crates/grammar/meta/syntax.toml:107]:
```suggestion
supported = true
```

File: crates/grammar/meta/syntax.toml [hunk 2]
```diff
@@ -134,10 +138,6 @@ supported = false
 name = "let"
 supported = true
 
-[[keywords]]
-name = "is"
```
Comment (saulecabrera) [inline crates/grammar/meta/syntax.toml:138]: I took this from https://github.com/AssemblyScript/assemblyscript/blob/main/src/tokenizer.ts#L236. That keyword is used for user-defined type guards https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates

Comment (saulecabrera) [reply crates/grammar/meta/syntax.toml:138]: So I believe we should have both

File: crates/grammar/meta/syntax.toml [hunk 3]
```diff
@@ -134,10 +138,6 @@ supported = false
 name = "let"
 supported = true
```
Comment (willemneal) [inline crates/grammar/meta/syntax.toml:140]:
```suggestion

[[keywords]]
name = "is"
supported = false

```

**Review by saulecabrera — APPROVED** [review]

**Pull request merged**